executor/show_stats.go

```diff
@@ -176,18 +176,20 @@ func (e *ShowExec) appendTableForStatsHistograms(dbName, tblName, partitionName
 		return
 	}
 	for _, col := range stableColsStats(statsTbl.Columns) {
-		// Pass a nil StatementContext to avoid column stats being marked as needed.
-		if col.IsInvalid(nil, false) {
+		if !col.IsStatsInitialized() || col.TotalRowCount() == 0 {
```

Comment:
When `col.TotalRowCount()` is zero, can we also display it? So we can distinguish between uninitialized stats and zero-count stats.